src/app/pages/home/components/onboarding-step-item.tsx

```diff
@@ -0,0 +1,125 @@
+import { FiArrowRight, FiCheck } from 'react-icons/fi';
+import { css } from '@emotion/css';
+import { Box, Circle, color, Stack } from '@stacks/ui';
+
+import { Body, Caption, Text, Title } from '@app/components/typography';
+import { Link } from '@app/components/link';
+import { isFullPage, isPopup } from '@app/common/utils';
+import { OnboardingSelectors } from '@tests/integration/onboarding.selectors';
+
+const fullPageStepMaxWidth = '200px';
+const popupPageStepWidth = '160px';
+const popupPageStepMaxHeight = '144px';
+const fullPageImageSize = '94px';
+const popupPageImageSize = '47px';
+
+interface StepIllustrationProps {
+  image: string;
+}
+const StepIllustration = ({ image }: StepIllustrationProps) => <img src={image} />;
+
+const PopupDoneIcon = () => (
+  <Circle
+    border="1px solid"
+    borderColor={color('border')}
+    color={color('bg')}
+    size={popupPageImageSize}
+  >
+    <FiCheck color={color('accent')} />
+  </Circle>
+);
+
+interface OnboardingStepItemProps {
+  body: string;
+  imageFull: string;
+  imagePopup: string;
+  isDone: boolean;
+  onClick(): void;
+  title: string;
+}
+export function OnboardingStepItem(props: OnboardingStepItemProps) {
+  const { body, imageFull, imagePopup, isDone, onClick, title } = props;
+
+  return (
+    <Stack
+      _hover={{
+        cursor: isPopup && !isDone ? 'pointer' : 'unset',
+      }}
+      border={['1px solid', 'unset']}
+      borderColor={color('border')}
+      borderRadius={['10px', 'unset']}
+      className={css`
+        @media (max-width: 442px) {
+          &:nth-child(2) {
+            height: 144px;
+          }
+        }
+      `}
```

Comment:
Yep, I can separate it.

Reply:
I have 2 comments about this, but nothing really actionable since I don't have the context:

- In general it's good to think about UI mobile first, meaning we define the styles for the smallest experience and we change it for larger viewports. therefore using `min-width` is always encourage unless there is an edge case where we must use `max-width`
- Why is the breakpoint hardcoded at `442px`? I am not saying that 442 is bad, I am just saying that it should probably be a variable that we can reuse in other place like `width-medium` for example so we don't have confusion around breakpoints. There is also an argument to define breakpoints in `rem` instead, but I don't want to get into the CSS rabbit whole because you'll never rescue me from there. It's so nice and cozy.

Reply:
Thx for the feedback, our breakpoints are defined in our imported theme from our UI library and have no relationship to our extension views. Here, this isn't being used for resizing the full page view, I need it to behave differently in our extension popups. I can def export that width from the `popup-center.ts` where the width of the center popup is defined. This should all work better together so that our breakpoints make sense with the extension views, but right now they don't. We can override the imported breakpoints ...I believe they are set here in the UI library here: https://github.com/hirosystems/ui/blob/c9efa75863b29b5f30302b38009dded893cf9b0c/packages/ui-theme/src/theme.ts#L26

Reply:
Nvm, I guess the breakpoints work fairly well using the array syntax, but not sure how to better handle it in this specific case.

Reply:
I ended up exporting the popup width and using a shared variable for the media query. I also relocated the style as was originally requested, let me know if there is more to change here?

Reply:
Thanks Fara. Unfortunately I am too far away from the code to understand the details of the implementation of the values of breakpoints. I hope my comment helped and not made things worst.

Maybe one day we can do a deep dive 🤔